helm/charts/infra/templates/server/configmap.yaml

```diff
@@ -61,6 +61,16 @@ data:
 
 {{- $users | uniq | toYaml | nindent 6 }}
 
+{{- if .Values.server.config.ui.proxyURL }}
+    ui:
+      proxyURL: {{ .Values.connector.config.ui.proxyURL }}
+{{- end }}
+
+{{- if and (include "ui.enabled" . | eq "true") (not .Values.server.config.ui.proxyURL ) }}
```

Comment:
Can group this with the previous condition to simplify it

```
{{- if .Values.server.config.ui.proxyURL }}
    ...
{{- else if include "ui.enabled" . | eq 'true' }}
    ...
{{- end }}
```

Reply:
Much better. Thanks!